common/src/main/scala/org/apache/celeborn/common/protocol/message/ControlMessages.scala

```diff
@@ -343,6 +343,10 @@ object ControlMessages extends Logging {
 
   case class CheckQuotaResponse(isAvailable: Boolean, reason: String) extends Message
 
+  case class CheckWorkersAvailable() extends Message
```

Comment:
Hi @SteNicholas , thanks for addressing the comments! Currently we prefer to use raw Pb messages  instead of defining case class (we define case objects for building Pbs). I recommend you to follow `RegisterWorker` and `RegisterWorkerResponse`. cc @pan3793 could you also take a look at this PR?

Reply:
@waitinfuture, @pan3793, I have followed the pattern of `RegisterWorker` and `RegisterWorkerResponse`. PTAL. BTW, could the standard that uses raw Pb messages instead of defining case class document in contribution page?

Reply:
> @waitinfuture, @pan3793, I have followed the pattern of `RegisterWorker` and `RegisterWorkerResponse`. PTAL. BTW, could the standard that uses raw Pb messages instead of defining case class document in contribution page?

Thanks, I've raised a PR describing this https://github.com/apache/incubator-celeborn/pull/1851